CHANGES.md

```diff
@@ -67,6 +67,9 @@
 - The test suite for Coq now requires Coq >= 8.16 due to changes in the
   plugin loading mechanism upstream (which now uses findlib).
 
+- Improve error message when the (name) of an executable is not part of the
+  (modules) listed for it. (#6169, fixes #5787, @emillon)
```

Comment:
FYI we don't usually document changes to error messages. Although of course I won't object if you do it.

Reply:
Right, I'm reverting that part.